CenterForOpenScience/osf.io
# [ENG-4447] Remove Depreciated Search Endpoints

## Purpose

Remove old search behavior, make way for new consolidated search.

## Changes

- removes endpoints: ( These have been checked to ensure they aren't being used )
  - `/explore/`
  - '/api/v1/search/projects/'
  - '/api/v1/search/users/'
  - '/api/v1/share/registration/'
  - '/api/v1/share/search/'
- removes confusing `SEARCH_ENGINE` config var. ⚠️ CONFIG CHANGES

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify all v1 search endpoints are safely removed

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects



## Ticket

https://openscience.atlassian.net/browse/ENG-4447
